src/coreclr/nativeaot/BuildIntegration/Microsoft.DotNet.ILCompiler.targets

```diff
@@ -14,5 +14,5 @@ Copyright (c) .NET Foundation. All rights reserved.
   <PropertyGroup>
     <NeedNativePublishSupportForSDK6>$(NETCoreSdkVersion.StartsWith('6'))</NeedNativePublishSupportForSDK6>
   </PropertyGroup>
- <Import Project="$(ILCompilerTargetsPath)" Condition="'$(NeedNativePublishSupportForSDK6)' == 'true'"/>
+  <Import Project="$(ILCompilerTargetsPath)" Condition="'$(NeedNativePublishSupportForSDK6)' == 'true'"/>
```

Comment:
`NeedNativePublishSupportForSDK6` can be deleted (separate PR). We do not need to keep it around anymore.